src/pages/library/Showcase.js

```diff
@@ -74,7 +77,14 @@ const WorkspaceCard = ({ workspace }) => {
     div({ style: { flex: 1, minWidth: 0, padding: '15px 20px', overflow: 'hidden' } }, [
       div({ style: { display: 'flex' } }, [
         div({ style: { flex: 1, color: colors.accent(), fontSize: 16, lineHeight: '20px', height: 40, marginBottom: 7 } }, [name]),
-        created && div([Utils.makeStandardDate(created)])
+        created && div([Utils.makeStandardDate(created)]),
+        (cloudPlatform === 'Azure' || cloudPlatform === 'Gcp') && h(CloudProviderIcon, {
```

Comment:
Validating `cloudPlatform` since showcase data is loaded from a JSON file in the firecloud-alerts bucket and thus may not be as stable as an API response. This also guards against `cloudPlatform` being undefined if someone uses an old version of the showcase.py script to update showcase.json.